internal/wrappers/features/collection.feature

```diff
@@ -0,0 +1,60 @@
+Feature: Collection management
+
+  Scenario: Create a collection
+    Given Kuzzle Server is running
+    And there is an index 'test-index'
+    When I create a collection 'collection-test-collection'
+    Then the collection should exists
+
+  Scenario: Check if a collection exists
+    Given Kuzzle Server is running
+    And there is an index 'test-index'
+    And it has a collection 'test-collection'
+    When I check if the collection exists
+    Then it should exists
+
+  Scenario: List existing collections
+    Given Kuzzle Server is running
+    And there is an index 'list-test-index'
+    And it has a collection 'test-collection1'
+    And it has a collection 'test-collection2'
+    When I list the collections
+    Then I should have 2 collections
+
```

Comment:
We could go further in the test:
Then the result contains 2 hits
And the list le list has 'test-collection1' and 'test-collection2'